fuzz_lightyear/plugins/idor.py

```diff
@@ -34,13 +34,13 @@ def is_vulnerable(
         request_sequence: List[FuzzingRequest],
         response_sequence: List[Any],
     ) -> bool:
-        last_request = request_sequence[-1]
-        try:
-            last_request.send(
-                auth=get_abstraction().get_attacker_session(),  # type: ignore
-                should_log=False,
-            )
-
-            return True
-        except (HTTPError, SwaggerMappingError, ValidationError):
-            return False
+        for request in request_sequence:
+            try:
+                request.send(
+                    auth=get_abstraction().get_attacker_session(),  # type: ignore
+                    should_log=False,
+                )
+
+            except (HTTPError, SwaggerMappingError, ValidationError):
+                return False
+        return True
```

Comment:
I'm unconvinced that this actually adds anything. Doesn't this just emulate a different user session, executing a completely tangential sequence of requests?